src/plays/download-fetch-response/CustomHooks/UseCustomFetchAPI.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react";
+
+export const UseCustomFetchAPI = (endPoint) => {
+  const [data, setData] = useState();
+  const [error, setError] = useState(false);
+  const [fetching, setFetching] = useState(true);
+
+  useEffect(() => {
+    const controller = new AbortController();
```

Comment:
A line of comments on `AbortController` will be good.

Reply:
Added